design/one-pager-default-resource-class.md

```diff
@@ -63,13 +63,13 @@ kind: ResourceClass
 metadata:
   name: cloud-postgresql
   namespace: crossplane-system
+  labels:
+    postgresql.storage.crossplane.io/default: "true"
```

Comment:
Should this be the actual kind, which I think is `postgresqlinstance` rather than `postgresql`?

Reply:
Yes! Good catch